packages/@aws-cdk/aws-apigateway/lib/domain-name.ts

```diff
@@ -2,7 +2,15 @@ import * as acm from '@aws-cdk/aws-certificatemanager';
 import { Construct, IResource, Resource } from '@aws-cdk/core';
 import { CfnDomainName } from './apigateway.generated';
 import { BasePathMapping, BasePathMappingOptions } from './base-path-mapping';
-import { EndpointType, IRestApi} from './restapi';
+import { EndpointType, IRestApi } from './restapi';
+
+/**
+ * The minimum version of the SSL protocol that you want Api Gateway to use for HTTPS connections.
+ */
+export enum SecurityPolicy {
+  TLS_1_0 = 'TLS_1_0',
+  TLS_1_2 = 'TLS_1_2'
```

Comment:
I would rather avoid the awslint exclusions in the `package.json`.

```suggestion
  /** Cipher suite TLS 1.0 */
  TLS_1_0 = 'TLS_1_0',
  /** Cipher suite TLS 1.2 */
  TLS_1_2 = 'TLS_1_2'
```

Reply:
TLS 1.0 is EOL on the march 31, 2020 ( ie, in 11 days) - so we should use 1.2 ?